package.json

```diff
@@ -5,13 +5,13 @@
     "eleventy": "eleventy",
     "html": "html-minifier --config-file=.html-minifier.json --input-dir=dist --output-dir=dist --file-ext=html",
     "css": "postcss src/_css/main.css --output dist/_css/main.css && postcss src/_css/feature-support.css --output dist/_css/feature-support.css",
-    "js": "mkdir -p dist/_js && minify --mangle.topLevel src/_js/main.mjs --outFile dist/_js/main.mjs && minify --mangle.topLevel src/_js/legacy.js --outFile dist/_js/legacy.js",
+    "js": "shx mkdir -p dist/_js && minify --mangle.topLevel src/_js/main.mjs --outFile dist/_js/main.mjs && minify --mangle.topLevel src/_js/legacy.js --outFile dist/_js/legacy.js",
     "json": "json-minify src/.webmanifest > dist/.webmanifest",
     "xml": "minify-xml --in-place dist/sitemap.xml && minify-xml --in-place dist/blog.atom && minify-xml --in-place dist/features.atom",
     "sw": "workbox injectManifest workbox-config.js && minify dist/sw.js --outFile dist/sw.js",
-    "copy-img": "for browser in chrome firefox; do cp ./node_modules/@browser-logos/${browser}/${browser}.svg src/_css/img/${browser}.svg; done; cp ./node_modules/@browser-logos/safari/safari_256x256.png src/_css/img/safari.png",
+    "copy-img": "shx cp ./node_modules/@browser-logos/{chrome,firefox}/*.svg src/_css/img/ && shx cp ./node_modules/@browser-logos/safari/safari_256x256.png src/_css/img/safari.png",
     "img": "node add-intrinsicsize.js",
-    "clean": "rm -rf -- dist",
+    "clean": "shx rm -rf dist",
```

Comment:
Does the `--` need to be removed?

Reply:
I don't think so / not sure, but it's not necessarily there either.

Reply:
Call me paranoid, but I'd prefer keeping it, in case we ever need to add `*` to the arguments.

Reply:
I mean... `*` would work too (why wouldn't it?). But ok, I can add it back.

Reply:
Fixed.